dashboard/src/components/AppUpgrade/AppUpgrade.tsx

```diff
@@ -95,13 +95,10 @@ function AppUpgrade({
     }
   }, [getDeployedChartVersion, app, chart, repoName, repoNamespace, cluster]);
 
-  if (appsError) {
-    return (
-      <Alert theme="danger">
-        An error occurred while processing the application: {appsError.message}
-      </Alert>
-    );
+  if (error && error.constructor === FetchError) {
```

Comment:
Just as an example, this could be:
```
if (errors.IsA(e, errors.FetchError) {
...
```
if we added that to the errors module for use. Again, not necessary, just mentioning.

Reply:
I prefer to avoid that syntactic sugar because it confuses the type checking. For example, I am defining `error: FetchError | UpgradeError`, if for some mistake I write `isA(e, DeleteError)` the compiler won't complain (but it will never be true) while if I do `e.constructor === DeleteError` I get the compilation error saying that the condition will never be true.